drivers/overlay/overlay.go

```diff
@@ -394,9 +394,11 @@ func parseOptions(options []string) (*overlayOptions, error) {
 			}
 		case "mount_program":
 			logrus.Debugf("overlay: mount_program=%s", val)
-			_, err := os.Stat(val)
-			if err != nil {
-				return nil, fmt.Errorf("overlay: can't stat program %s: %v", val, err)
+			if val != "" {
+				_, err := os.Stat(val)
+				if err != nil {
+					return nil, fmt.Errorf("overlay: can't stat program %s: %v", val, err)
```

Comment:
non-blocking nit since we're here
```suggestion
					return nil, errors.Wrapf(err, "overlay: can't stat program %s", val)
```

Reply:
thanks!  I've added a new patch with:

```
				return nil, errors.Wrapf(err, "overlay: can't stat program %q", val)
```